core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -316,6 +333,44 @@ public ServerBuilder port(ServerPort port) {
         return this;
     }
 
+    /**
+     * Adds a {@link ChannelOption} of {@link Server}'s server sockets.
+     * Note that the previously added option will be overridden if the same option is set again.
+     *
+     * <pre>{@code
+     * ServerBuilder sb = new ServerBuilder();
+     * sb.serverChannelOption(ChannelOption.BACKLOG, 1024)
+     *   .serverChannelOption(ChannelOption.SO_KEEPALIVE, true);
+     * }</pre>
+     */
+    public <T> ServerBuilder serverChannelOption(ChannelOption<T> option, T value) {
+        requireNonNull(option, "option");
+        checkArgument(!PROHIBITED_SOCKET_OPTIONS.contains(option),
+                      "prohibited socket option: %s", option);
+
+        this.serverChannelOptions.put(option, value);
+        return this;
+    }
+
+    /**
+     * Adds an {@link ChannelOption} of {@link Server}'s server child sockets .
+     * Note that the previously added option will be overridden if the same option is set again.
+     *
+     * <pre>{@code
+     * ServerBuilder sb = new ServerBuilder();
+     * sb.serverChannelOption(ChannelOption.BACKLOG, 1024)
```

Comment:
BACKLOG is only useful for server sockets. How about using something else?

Reply:
You also need to update the example to use the setter that updates the child option.